recipe.cake

```diff
@@ -109,7 +109,7 @@ Task("Prepare-Chocolatey-Packages")
     // Run Chocolatey Unpackself
     CopyFile(BuildParameters.Paths.Directories.PublishedApplications + "/choco_merged/choco.exe", BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe");
 
-    StartProcess(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe", new ProcessSettings{ Arguments = "unpackself -f --allow-unofficial-build" });
+    StartProcess(BuildParameters.Paths.Directories.ChocolateyNuspecDirectory + "/tools/chocolateyInstall/choco.exe", new ProcessSettings{ Arguments = "unpackself -f -y --allow-unofficial-build" });
```

Comment:
I don't think that this would bypass the non-admin warning? It still shows the non-admin warning even with `--yes` passed for me.
![image](https://user-images.githubusercontent.com/38865330/175369038-5cc01f53-f04f-4306-9c1b-f7cb1be1fe3e.png)